tests/containers/docker_compose.pm

```diff
@@ -37,7 +37,13 @@ sub run {
 
     record_info 'Test #1', 'Test: Installation';
 
-    my $ret = zypper_call "in docker-compose", exitcode => [0, 4];
+    my $ret;
+    if (is_sle || is_leap) {
```

Comment:
maybe to avoid potential errors in other platforms, I would have done 
`if is_tumbleweed` -> install switch.
`else`,  leave default.

Reply:
`is_tunbleweed` is something that should be avoided (see the code style policy) and IMO it does make sense here. At some point (15.5?) Leap and SLE will also get the change, then you just need to add a version number.